raddar/core/contexts.py

```diff
@@ -9,7 +9,6 @@
 
 
 @contextmanager
-# def clone_repo(project_name) -> ContextManager[Repo]:
 def clone_repo(project_dir: str, project_name: str, ref_name: str):
```

Comment:
Do you think you could type the return value as `ContextManager[Tuple[Repo, str]]`?